data-model/src/main/java/ch/prevo/open/node/data/provider/MatchNotificationListener.java

```diff
@@ -1,12 +1,12 @@
 package ch.prevo.open.node.data.provider;
 
-import ch.prevo.open.encrypted.model.CommencementMatchNotification;
-import ch.prevo.open.encrypted.model.TerminationMatchNotification;
+import ch.prevo.open.data.api.FullCommencementNotification;
+import ch.prevo.open.data.api.FullTerminationNotification;
 
 public interface MatchNotificationListener {
 
-    void handleTerminationMatch(TerminationMatchNotification notification);
+    void handleTerminationMatch(FullTerminationNotification notification);
```

Comment:
i.e. throw an exception?

Reply:
Good idea, but I'd rather put it into another issue. This would require changes in several places, e.g. the communication between hub and node, the matching algorithm.

Reply:
ok - could you then create a new issue and link it here?

Reply:
done #88